tmail-backend/jmap/extensions/src/main/scala/com/linagora/tmail/james/jmap/method/CustomMethod.scala

```diff
@@ -17,7 +17,6 @@ object CapabilityIdentifier {
   val LINAGORA_ECHO: CapabilityIdentifier = "com:linagora:params:jmap:echo"
   val LINAGORA_FILTER: CapabilityIdentifier = "com:linagora:params:jmap:filter"
   val LINAGORA_PGP: CapabilityIdentifier = "com:linagora:params:jmap:pgp"
-  val LINAGORA_LONG_LIVED_TOKEN: CapabilityIdentifier = "com:linagora:params:long:lived:token"
```

Comment:
Can you remove it also in `docs/modules/ROOT/pages/tmail-backend/jmap-extensions/index.adoc`